vignettes/geotargets.Rmd

```diff
@@ -41,15 +41,21 @@ In this vignette we will demonstrate the use of the `terra` R package, and we wi
 -   `tar_terra_sprc()`
 -   `tar_terra_sds()`
 
+# How to run targets examples from vignettes
+
+The way targets typically works is you write a file named `_targets.R`, which describes the pipeline. See for example, the [`_targets.R` file in the demo-geotargets repo](https://github.com/njtierney/demo-geotargets/blob/main/_targets.R).
+
+However, in order to demonstrate many of the features with `targets` and `geotargets`, we don't want to have to create many `_targets.R` files. So instead we use a targets function [`targets::tar_script()`](https://docs.ropensci.org/targets/reference/tar_script.html). This allows you to write the code you would have put in a `_targets.R` file. 
+
+What this means for you is you can essentially just "copy and paste" the examples we provide in this vignette. When running the `tar_script` code, it will ask you each time if you want to overwrite the `_targets.R` file. This means if you are exploring these examples and copying the entire examples, it is worthwhile doing this in a separate repository to avoid overwriting your own `_targets.R` file.
+
```

Comment:
I might flip the order here.  Something like:

>The example code in this vignette is designed for you to be able to just copy and paste into the R console.  However, this is not a typical way to run a `targets` workflow.  The examples make use of `targets::tar_script()` which creates or overwrites a `_targets.R` file.  In these examples, everything inside of `tar_script({})` is what would go inside a `_targets.R` file defining a workflow. When building your own `targets` workflow it is not recommended to use `tar_script()` but rather to edit `_targets.R` directly.

Reply:
Also, is it OK (e.g. according to CRAN) to use `tar_script()` without also wrapping it in `tar_dir()`?

Reply:
Not sure about the CRAN comment here, perhaps we shall have to find out!